### ISSUE TITLE
Examples not correct/present

### ISSUE THREAD

**Issue by ethanmsmith** [MicrosoftDocs/office-docs-powershell#1299]
I think the examples could use some work. They don't seem to run in PowerShell...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0537d84c-b9ac-01d9-09cb-70c8eafa37cc
* Version Independent ID: 61263d32-fb5e-7e74-ed87-0ee6f855ecca
* Content: [New-ComplianceRetentionEvent (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/policy-and-compliance-retention/new-complianceretentionevent?view=exchange-ps#feedback)
* Content Source: [exchange/exchange-ps/exchange/policy-and-compliance-retention/New-ComplianceRetentionEvent.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/policy-and-compliance-retention/New-ComplianceRetentionEvent.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@ethanmsmith, there appears to be a lot in this topic that isn't complete. I'll get things started.

Thanks for the feedback.

**Comment by AndreaBarr** [comment]
@ethanmsmith Thank you for this feedback

I am going to get this send to the PowerShell Backlog for tracking and have them comment back when they have an update. 

I was able to get this into the backlog for the PowerShell team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
